src/ui/public/chrome/api/__tests__/nav.js

```diff
@@ -45,6 +45,19 @@ describe('chrome nav apis', function () {
     });
   });
 
+  describe('#getNavLinkById', () => {
+    it ('retrieves the correct nav link, given its ID', () => {
```

Comment:
There should be a test for the case where the id is not found.  If possible, it'd be great if we could throw in that case, what do you think?